lib/si-crypto/src/cyclone/encryption_key.rs

```diff
@@ -59,6 +85,30 @@ impl CycloneEncryptionKey {
         })
     }
 
+    /// Loads a [`CycloneEncryptionKey`] from a base64 encoded string.
+    ///
+    /// # Errors
+    ///
+    /// Return `Err` if:
+    ///
+    /// - A key string could not be successfully parsed
+    pub async fn decode(
+        encryption_key_string: String
+    ) -> Result<Self, CycloneEncryptionKeyError> {
+        trace!(
+            "loading cyclone encryption key from base64 string {}", encryption_key_string
+        );
+        let buf = general_purpose::STANDARD.decode(encryption_key_string).map_err(CycloneEncryptionKeyError::Base64Decode)?;
```

Comment:
I think we should check if the last line is \n here and prune it if it is. It'd save us the hassle of having to (or forgetting to) `echo -e <content> | bas64` when placing the values into secrets manager